README.md

```
@@ -153,13 +153,13 @@
 | `make up`        | 🐳 Starts all development containers in detached mode.                   |
 | `make down`      | 🛑 Stops and removes all development containers.               |
 | `make logs`      | 📜 Tails the logs of the API service in real-time.             |
-| `make shell`     | 💻 Opens an interactive shell (`/bin/sh`) inside the API container.|
+| `make shell`     | 💻 Opens an interactive shell (`/bin/sh`) inside the API container.||
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)